otx/cli/tools/train.py

```diff
@@ -195,9 +216,14 @@ def main():
         task = run_hpo(args, environment, dataset, template.task_type)
         if task is None:
             print("cannot run HPO for this task. will train a model without HPO.")
-            task = task_class(task_environment=environment)
+            task = task_class(task_environment=environment, output_path=args.save_logs_to)
```

Comment:
So, we have logs and checkpoints in the args.save_logs_to, is it right?

Reply:
yes. you're right.

Reply:
If so, args.save_logs_to should change name such like args.output_path. In addition what if someone want to separate output log path and weight path, can this code handle it?

Reply:
Although I just used already made argument `args.save_logs_to`, but I agree with your opinion. I'll change its' name to `args.output_path`.
Do you mean that weight file is final model weight? then, user can set by using `--save-model-to` argument.

Reply:
Then a user gives both save_logs_to and save-model-to, then are log and model weight saved in separate directory?